sqldelight-runtime/src/main/java/com/squareup/sqldelight/EnumColumnAdapter.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Copyright (C) 2016 Square, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.squareup.sqldelight;
-
-import android.support.annotation.NonNull;
-
-/** A {@link ColumnAdapter} which maps the enum class {@code T} to a string in the database. */
-public final class EnumColumnAdapter<T extends Enum<T>> implements ColumnAdapter<T, String> {
```

Comment:
worth calling out that this column adapter is impossible to do generically with pure kotlin because `enumValueOf<T>` requires a reified type. I plan on including this class back in the `sqldelight-android-runtime` artifact

Reply:
https://youtrack.jetbrains.com/issue/KT-14743

Reply:
watched that issue anyways to see how it develops

Reply:
Require them to pass in `Array<out T>` via `enumValuesOf<Whatever>()`?

Reply:
nice i like that